src/game/Player.cpp

```diff
@@ -1039,18 +1039,48 @@ void ARX_PLAYER_Restore_Skin() {
 			tx4 = "graph/obj3d/textures/npc_human_leather_hero4_head";
 			break;
 		case 4:
-			tx  = "graph/obj3d/textures/npc_human_cm_hero_head";
-			tx2 = "graph/obj3d/textures/npc_human_chainmail_hero_head";
-			tx3 = "graph/obj3d/textures/npc_human_chainmail_mithril_hero_head";
-			tx4 = "graph/obj3d/textures/npc_human_leather_hero_head";
+			tx  = "graph/obj3d/textures/npc_human_base_hero5_head";
+			tx2 = "graph/obj3d/textures/npc_human_chainmail_hero5_head";
+			tx3 = "graph/obj3d/textures/npc_human_chainmail_mithril_hero5_head";
+			tx4 = "graph/obj3d/textures/npc_human_leather_hero5_head";
 			break;
 		case 5:
-			tx  = "graph/obj3d/textures/npc_human__base_hero_head";
+			tx  = "graph/obj3d/textures/npc_human_base_hero6_head";
```

Comment:
would it make sense to have it like this?
`
case 5:

case 6:

case 7:
.....
tx = "the string with the number inserted";
tx2= ...
`

cuts down on repetition